Documentation/getting-started.md

```diff
@@ -27,23 +28,41 @@ The Adobe Experience Platform Identity for Edge Network extension has the follow
   $ pod install
   ```
 
-3. Import the Mobile Core and Edge libraries:
+3. Import the Mobile Core and Edge libraries and register Edge Extension with MobileCore:
 
+#### Swift
   ```swift
   // AppDelegate.swift
   import AEPCore
   import AEPEdge
   import AEPEdgeIdentity
   ```
 
-4. Register the Identity for Edge Extension with MobileCore:
-
   ```swift
```

Comment:
nit: As these changes are in the same file, you can combine them both in the same "swift" language tag and just separate with "...".

Reply:
We kept it separate so that the developer can copy paste it separately.